content/en/security/application_security/threats/threat-overview.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Threat Overview
+kind: documentation
+aliases:  
+---
+
+{{< img src="security/application_security/threats/appsec-threat-overview-page-top.png" alt="Screenshot of the ASM Threat Overview page"  >}}
+
+## Introduction
+
+The ASM Threats Overview provides an at a glance view of your application and API posture, highlighting trends, service exposure, attack traffic, and the impact on business logic. Users can quickly pivot from widgets to related traces.
+
+The Threats Overview is divided into several sections, each focusing on a different aspect of security with supporting information.
+
+## Sections
+
+**Attack Surface Area**: This section provides insights into the exposed services, the tools attackers are using, and the commercial scanners that are identifying potential vulnerabilities.
+
+**Attack Traffic**: These graphs identify the classification of attacks, such as SSRF, LFI, SQL and command injection, allowing users to identify spikes in malicious traffic and patterns.
+
+**Business Logic**: This section focuses on the application-specific operations, such as account takeover attempts and custom business logic metrics, to help you understand trending attacks related to user accounts or fraud.
+
+**Geographic Overview**: A global heatmap indicating the sources of attack traffic, providing a visual representation of threats by region.
+
+## Best Practices
+
+1. Users should review trends and adopt a corresponding protection policy to meet their posture needs.
+2. Regularly review the Exposed Services widget to ensure only intended services are accessible and that they have a protection policy that meets your risk profile.
+3. Block attack tools and ensure that customer scanners are part of an authorized vulnerability management program.
+4. Monitor business logic for changes in detection profiles related to distributed credential stuffing campaigns or risky payment activity.
```

Comment:
Maybe what I put below is too simplistic but `changes in detection profiles` is a blurry notion for me.
4. Motinor business logic for spikes in credential stuffing attacks or risky payment activity

Reply:
+2